templates/simulate.html

```diff
@@ -14,11 +14,17 @@
             border-radius: 25px 25px 25px 25px;
             width: 150px;
```

Comment:
consider chaning file name, it is confusing with the actuals simulation's name, consider maybe simulation-input